sevsnpmeasure/cli.py

```diff
@@ -12,7 +12,7 @@
 from sevsnpmeasure import vmm_types
 from .sev_mode import SevMode
 
-VERSION = '0.0.7'
+VERSION = '0.0.8'
```

Comment:
no, I'll do that separately. Please remove.
I might release a version before merging this and then after.